src/prr.rs

```diff
@@ -37,6 +64,23 @@ pub struct Prr {
     config: Config,
     /// Instantiated github client
     crab: Octocrab,
+    /// Local config
+    local: Option<PrrLocalConfig>,
+}
+
+/// Returns if exists the config file for the current project
+fn find_project_config_file() -> Option<PathBuf> {
+    env::current_dir().ok().and_then(|mut path| loop {
+        path.push(CONFIG_RC_FILE_NAME);
+        if path.exists() {
+            return Some(path);
+        }
+
+        path.pop();
+        if !path.pop() {
+            return None;
+        }
+    })
```

Comment:
How about calling this from main() and passing in an `Option<&Path>` into `Prr::new`. So that FS walking logic stays close to each other

Reply:
I'm not really sure about this edit. You're suggesting to pass the path at

```rust
let prr = Prr::new(&config_path)?;
```

changing with something like

```rust
let local_config = find_project_config_file();
let prr = Prr::new(&config_path, &local_config)?;
```

Reply:
Yeah